infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -297,6 +297,11 @@ type DefaultConfigs struct {
 	Image *string `json:"image,omitempty"`
 }
 
+type StoreServiceConfigs struct {
+	Replicas       *int32 `json:"replicas,omitempty"`
```

Comment:
this new field needs a description, something explaining to the user what it affects. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#deploymentspec-v1-apps

do we want to encourage the user to use persistence if setting above 1 ?